data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/key/rotation/IndividualKeyRotationResponse.avsc

```diff
@@ -27,7 +27,7 @@
     {
       "name": "targetKeyAlias",
       "type": "string",
-      "doc": "Specifies the wrapped key to rotate"
+      "doc": "Specifies the alias of the key that should be rotated"
```

Comment:
Let's add dot at the end of the sentence.

Reply:
done